projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -79,8 +97,14 @@ function shouldClose(
             useExisting: TuiHostedDropdownComponent,
         },
     ],
+    host: {
+        '[$.class._hosted_dropdown_focused]': 'focused$',
+        '($.class._hosted_dropdown_focused)': 'focused$',
+    },
 })
 export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
+    private readonly focus$ = new Subject<boolean>();
```

Comment:
Just make it `BehaviorSubject(false)` and next `this.focused` to it.

Reply:
fixed